test/features/user.admin.feature

```diff
@@ -73,8 +73,6 @@ Feature: User
     And I press "Create new account"
     Then I should see "Created a new user account for tempuser."
     When I am on "Users" page
-    And I fill in "edit-name" with "tempuser"
```

Comment:
Thanks for this! I got a conflict there and I was not sure about which change was the correct one. I already fixed this. Thanks!